clusters/minikube/configure.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+SYSTEM_INSTALL_FLAGS="${SYSTEM_INSTALL_FLAGS:---node-port}"
+
+wait_for_ingress_ready() {
+  name=$1
+  namespace=$2
+
+  # nothing to do
+}
+
+# Install minikube cli
+source `dirname "${BASH_SOURCE[0]}"`/../../install.sh minikube
```

Comment:
same, what about `./` instead?

Reply:
The tool will be installed with a sub shell, we can avoid the extra sub shell by sourcing.